projects/synapse/index.js

```diff
@@ -0,0 +1,91 @@
+const { chainExports } = require("../helper/exports");
+const { request, gql } = require("graphql-request");
+const { getBlock } = require("../helper/getBlock");
+const { default: axios } = require("axios");
+const sdk = require("@defillama/sdk");
+const retry = require("async-retry");
+const ethers = require("ethers");
+
+const gqlQuery = gql`
+  query getTVL($block: Int) {
+    swaps(block: { number: $block }) {
+      tokens(orderBy: symbol, orderDirection: asc) {
+        id
+        name
+        decimals
+      }
+      balances
+    }
+  }
+`;
+
+const addBalanceWithExisting = (balances, token, balance) => {
+  const _balances = {};
+
+  if (balances[token] !== undefined) {
+    balance = ethers.utils.bigNumberify(balances[token]).add(balance);
```

Comment:
Before I was conveting all UNKNOWN coins to USDC, so without that code it would throw an error. However I do not do that anymore so it is redundant code now, will remove and it doesnt dupe the values.